app/serializers/api/v1/stockit_item_serializer.rb

```diff
@@ -12,6 +12,7 @@ class StockitItemSerializer < ApplicationSerializer
     has_many :package_actions, serializer: PackageActionsSerializer, root: :item_actions
     has_one :storage_type, serializer: StorageTypeSerializer
     has_one :package_set, serializer: PackageSetSerializer::StockFormat
+    has_many :messages, serializer: MessageSerializer, polymorphic: true
```

Comment:
this can really slow down the serializer as it is used in package search. do we need it here? @swatijadhav

Reply:
With work on `notifications` this can now be removed.